src/handlebars/helpers.ts

```diff
@@ -14,9 +14,25 @@ export function registerHandlebarsHelpers() {
     }
     return options.inverse(this);
   });
+  handlebars.registerHelper(
+    'isCellInWinningPath',
+    function (
+      winningPath: Coordinates[],
+      cellXCoord: number,
+      cellYCoord: number,
+    ) {
+      return (
+        winningPath &&
+        winningPath.filter(
```

Comment:
do you know ythe `array.some()` function?

Reply:
Nop!